vesuvius-c.h

```diff
@@ -152,6 +174,7 @@ static int fetch_metadata(const char *url, char *buffer) {
 
 // Parses the metadata JSON to retrieve chunk sizes and shape
 static int parse_metadata(const char *buffer) {
+#if 0
```

Comment:
fix before merge 

This branch doesn't have the json fix so I'm doing a bunch of comment shenanigans right now

Reply:
addressed